### PR TITLE
Feat: ASC 00000  added user profile gallery

### PR DESCRIPTION
**Pull request by NaingAmity**

### CODE REVIEW

File: src/components/PollSection/usePoll.ts [hunk 1]
```diff
@@ -9,7 +9,10 @@ export const usePoll = (pollId: string, shouldFetch: boolean) => {
     if (pollData?.closedAt) {
       const endDate = dayjs(pollData.closedAt);
       const daysRemaining = endDate.diff(dayjs(), 'day');
-      return daysRemaining || 0;
+      if (daysRemaining > 0) return daysRemaining;
+      const hoursRemaining = endDate.diff(dayjs(), 'hour');
+      if (hoursRemaining > 0) return 1;
+      return 0;
```
Comment (frankent) [inline src/components/PollSection/usePoll.ts:15]: Why not just return `hoursRemaining`

```suggestion
      if (daysRemaining > 0) return daysRemaining;
      const hoursRemaining = endDate.diff(dayjs(), 'hour');
      return hoursRemaining > 0 ? hoursRemaining : 0
```

**Review by frankent — COMMENTED** [review]
1 question

File: src/components/PollSection/usePoll.ts (hunk 1 above)
Comment (NaingAmity) [reply src/components/PollSection/usePoll.ts:15]: UI is showing only days in figma xD

Comment (frankent) [reply src/components/PollSection/usePoll.ts:15]: it means if `hoursRemaining = 2` hr in UI will show as 1 day, am I correct ?

Comment (NaingAmity) [reply src/components/PollSection/usePoll.ts:15]: yes. exactly.

**Review by frankent — APPROVED** [review]
LGTM

**Pull request merged**